man/subscription-manager.8

```diff
@@ -426,6 +426,12 @@ The
 .B list
 command lists all of the subscriptions that are compatible with a system. The options allow the list to be filtered by subscriptions that are used by the system or unused subscriptions that are available to the system.
 
+.TP
+.B --after
+Shows pools that are active on or after the given date. This is only used with the
+.B --available
+option.
```

Comment:
Please add here information about format of the date used as argument of `--after` option. Something like: `YYYY-MM-DD`.